src/plugins/python/scripts/lib/utils.py

```diff
@@ -0,0 +1,17 @@
+import os.path
+
+
+def find_sensitive_path(dir, insensitive_path):
```

Comment:
What is a sensitive or insensitive path?

Reply:
case-in/sensitive, since DOS-based systems don't care about cases but *nix does. script should find case-sensitive path by using incorrect casing.